src/commands/stop.ts

```diff
@@ -0,0 +1,56 @@
+import chalk from 'chalk';
+import BaseCommand from "../base-command";
+import { DockerComposeUtils } from "../common/docker-compose";
+import inquirer from 'inquirer';
+import { DockerInspect } from '../common/docker-compose/template';
+
+export default class StopLocalDeployment extends BaseCommand {
+  async auth_required(): Promise<boolean> {
+    return false;
+  }
+
+  static description = 'Stop a local deployment';
+  static examples = [
+    'architect stop <local-environment-name>',
+  ];
+
+  static flags = {
+    ...BaseCommand.flags,
+  };
+
+  static args = [{
+    sensitive: false,
+    name: 'name',
+    description: 'Name of local environment',
+    required: false,
+  }];
+
+  async run(): Promise<void> {
+    const { args } = await this.parse(StopLocalDeployment);
+
+    const local_env_map = await DockerComposeUtils.getLocalEnvironmentContainerMap();
+    const env_names = Object.keys(local_env_map);
```

Comment:
You can use `DockerComposeUtils.getLocalEnvironments()` if you only need the keys (the `getLocalEnvironments` function is just `return Object.keys(await this.getLocalEnvironmentContainerMap());`)